desktop_server/src/com/mygdx/game/server/network/handlers/ConnectHandler.java

```diff
@@ -45,6 +46,8 @@ public void handle(String[] commands, Client client) {
               .map(Client::mapToPlayerLobby).collect(Collectors.toList());
       var msg = new PlayerJoinedRoomMessage(users);
       sender.sendToAll(msg, room.getClients());
+      var msg2 = new RoomConfigMessage(room.getMapSize());
+      sender.sendToAll(msg2, room.getClients());
```

Comment:
na pewno za każðym razem jak sie doda klient to dobrze jest wysyłać te dane?

Reply:
Napewno trzeba wysyłać config, żeby gracz wiedział jaki jes rozmiarmapy, ale nie potrzeba do wszystkich wysyłać, zmienię to